### ISSUE TITLE
Not working with atom project of several git repos

### ISSUE THREAD

**Issue by jgburet** [Axosoft/GitKraken-Bisect-Atom#4]
_Sorry, no issue template_

We are using multi repositories in my actual company, so everything is open in a single Atom Project.
When I try to start GK Bisect, I get a yellow flag informing me I'm not in a git repository.

Wouldn't it be nice to have it to work without having to open/create a new project only with a single repo?
No idea on how to propose the user which folder he wants to work with though :upside_down_face:

**Comment by implausible** [comment]
It would be fantastic!

**Comment by implausible** [comment]
I think what we had discussed working on was, in multiple repo situations, we should have some sort of drop down to select which repo we're operating on before starting the bisect. I'm not sure if that should be permanent UI at the the top that is disabled out once a bisect is started, and free for switching prior to starting a bisect.
